subprojects/core/src/main/java/org/gradle/api/internal/artifacts/dependencies/AbstractModuleDependency.java

```diff
@@ -163,6 +163,7 @@ protected void copyTo(AbstractModuleDependency target) {
         if (moduleDependencyCapabilities != null) {
             target.moduleDependencyCapabilities = moduleDependencyCapabilities.copy();
         }
+        target.endorsing = endorsing;
```

Comment:
❔ If this is included when copying, should it be added to `isCommonContentEquals` as well?

Reply:
Good catch! I think it should be. Will update this.